src/main/kotlin/org/jetbrains/kotlinx/jupyter/repl/creating/ReplFactory.kt

```diff
@@ -69,6 +81,17 @@ abstract class ReplFactory {
     protected val commManager: CommManager by lazy { provideCommManager() }
     protected abstract fun provideCommManager(): CommManager
 
+    protected val bundledCommHandlers: List<CommHandler> by lazy { provideCommHandlers() }
+    protected abstract fun provideCommHandlers(): List<CommHandler>
```

Comment:
Maybe just `commHandlers`? At least, property `<x>` should have method `provide<X>`, please make them match